tests/test_process_data.py

```diff
@@ -24,6 +24,7 @@ def test_count_elements(count_mode: ElemCountMode, counts: dict[str, float]) ->
     pd.testing.assert_series_equal(series, expected, check_dtype=False)
 
 
+# DEBUG: this test is failing in CI (not locally)
```

Comment:
I totally have no idea why this unit test is failing (in CI only, passed locally):
```python
def test_count_elements_by_atomic_nums() -> None:
    series_in = pd.Series(1, index=range(1, 119))
    el_cts = count_elements(series_in)
    expected = pd.Series(1, index=df_ptable.index, name="count")

    pd.testing.assert_series_equal(expected, el_cts)
```

As `ptable.process_data` is not touched during this PR at all...